### DISCUSSION TITLE
14 febbraio 2023 A - esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#29] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/d0cf1d2c-29f2-4aac-8b53-419893e22b30)

**Comment by MattiaGio** [comment]
Scusate la scrittura purtroppo è stato scritto a mano.
PS: Ho mandato una mail al professore e la sua risposta è stata che la soluzione "gli sembra corretta"
![photo_2024-01-13_20-41-26](https://github.com/sapienzastudentsnetwork/bdi2324/assets/104901925/a2aaa338-a651-47df-bd59-c19f35ddb481)

![photo_2024-01-13_20-41-39](https://github.com/sapienzastudentsnetwork/bdi2324/assets/104901925/4c5a1b23-80c9-44d9-bfc0-88d054745598)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 09:46:38](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/5a78049e-10fa-4c09-badc-ea0790763efc)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/338ddb17-83ec-443b-8138-d9c9b1a6aa1c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e6ad7160-498e-44f5-9cc6-9f8736c60363)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/3d804fbd-810c-4c23-8c4b-51639c7c4cf1)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/91b5af59-de0c-405b-9ede-f6c771aa4c6d)
